lib/Service/Attachment/AttachmentService.php

```diff
@@ -363,6 +364,20 @@ private function handleCloudAttachment(Account $account, array $attachment): ?in
 			return null;
 		}
 
+		$storage = $file->getStorage();
+		if ($storage->instanceOfStorage(SharedStorage::class)) {
+
+			/** @var SharedStorage $storage */
+			$share = $storage->getShare();
+			$attributes = $share->getAttributes();
+
+			if($attributes->getAttribute('permissions', 'download') === false) {
+				$this->logger->warning("Could not create attachment, no download permission for file: ${$attachment['fileName']}");
```

Comment:
```suggestion
				$this->logger->warning("Could not create attachment, no download permission for file: " . $attachment['fileName']);
```

Array accesses inside string interpolations are not working.